cli/tests/integration/general_tests.rs

```diff
@@ -0,0 +1,1036 @@
+// Copyright 2018-2021 the Deno authors. All rights reserved. MIT license.
```

Comment:
Nitpick: I suggest to leave these tests in `integration/mod.rs`

Reply:
Good idea! a3e407c